libPhoneNumber/NBGeocoderMetadataHelper.h

```diff
@@ -0,0 +1,52 @@
+//
+//  NBGeocoderMetadataHelper.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <sqlite3.h>
+
+@class NBPhoneNumber;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NBGeocoderMetadataHelper : NSObject
+
+/**
+* Initializer method that creates a SQLite3 connection to the correct database (based on
+* the language passed in as a parameter), which is stored in the resource bundle of this project.
+* This method opens the database for viewing if possible and prepares a query statement with
+* the country code parameter
+*
+* @param countryCode a valid country code for which we want to set up a prepared statement to
+*  access a SQLite table with that country code.
+* @param languageCode  the language code for which the description should be written
+* @return  a NBGeocoderMetadataHelper instance variable with an initial country code and language
+code set to the parameters, countryCode and  languageCode.
+*/
+- (instancetype)initWithCountryCode:(NSNumber *)countryCode withLanguage:(NSString *)languageCode;
+
+/**
+* Returns a text description for the given phone number. The description will be based on the
+country code
```

Comment:
Over 100 cols? Make sure no code/comment is over 100 cols.

Reply:
fixed